server/repositories/prisonApi.js

```diff
@@ -0,0 +1,93 @@
+const querystring = require('querystring');
+const { formatISO } = require('date-fns');
+const { logger } = require('../utils/logger');
+const { getEnv } = require('../../utils');
+
+class PrisonApiRepository {
+  constructor({ client, apiUrl }) {
+    this.client = client;
+    this.url =
+      apiUrl ||
+      getEnv('PRISON_API_BASE_URL', 'https://api.nomis', {
```

Comment:
do you need this defaulting using env vars or do you always pass a value?

Reply:
I've just updated this to require the environment variable to be set, otherwise we throw. I think we should be explicit in our configuration to avoid any mishaps 🤔